src/data/constants.js

```diff
@@ -0,0 +1,3 @@
+export const moviePageMainTitle = "Popular Movies";
```

Comment:
Please use uppercase letters with underscores for constants 
`MOVIE_PAGE_MAIN_TITLE`